src/components/browse/singleDatasetView/singleDatasetView.tsx

```diff
@@ -80,7 +80,7 @@ const SingleDatasetView = () => {
 
   return (
     <div className="py-2 py-sm-4 mx-auto px-2 px-sm-5">
-      {details === undefined ? (
+      {details === undefined || datasetFiles === undefined ? (
```

Comment:
The idea was actually to allow rendering the page before the datasetFiles are fetched, and then re-render when the remaining data arrives. For whatever reasons, the re-rendering does not seem to work, right? This fixes it, and it is good enough for the legacy app, but in the new app we should do such things properly.

Reply:
It's because the table data is not generated or updated asynchronously yes. And yes, this is just a simple patch.